src/comment_worker.py

```diff
@@ -685,7 +694,93 @@ def promote(self, sess, comment, investor, to_promote):
                 REDDIT.subreddit(subreddit).flair.set(user.name, f"{firm.name} | {flair_role_user}")
                 REDDIT.subreddit(subreddit).flair.set(investor.name, f"{firm.name} | {flair_role_investor}")
 
-        return comment.reply_wrap(message.modify_promote(user))
+        return comment.reply_wrap(message.modify_promote(user, user_role))
+
+    @req_user
+    def demote(self, sess, comment, investor, to_demote):
```

Comment:
Can you also add some comments in your code? I am not asking for docstrings or official Sphinx kind of documentation. Just inline comments. For example,

``` python
if user_role == "":
 ...
```

Can be turned into 

``` python
# If investor is already at the lowest rank (floor trader), we can't demote him.
if user_role == "":
 ...
```

And something like that. Currently, I plan to sprinkle the code with inline commenting and docstrings so we can start forming an official documentation for the code. Thanks

Reply:
Also yeah, depending on your IDE, if you have a big/long comment, break it down to several lines. Usually every comment line is no longer than 80 characters.

Reply:
``` python
# This is a multiline comment. This is a very big comment that looks weird so
# I have to break it down. Thank you for your attention.